docs/adr/0007-wikibase-release-notes-publish.md

```diff
@@ -0,0 +1,61 @@
+# 7) Wikibase Extension release notes publishing {#adr_0007}
+
+Date: 2021-01-14
+
+## Status
+
+proposed
+
+## Context
+
+We have in [ADR0005] proposed a process for maintaining our release notes within the repository we also want to make a decision as to where these should be published for a broader audience.
+
+As our process for maintaining release notes will be closely modeled after mediawiki it also makes sense to review how mediawiki is publishing theirs.
+
+As described in [ADR0005], the mediawiki/core team is maintaining release notes for each specific release within the corresponding release branch. During the preparation of creating a new release from the development branch, a new release branch is cut and the [template] that has been populated with the relevant changes from the ongoing development is to serve as the base for a richer curated version on mediawiki.org [(Example: 1.35)](https://www.mediawiki.org/wiki/Release_notes/1.35).
```

Comment:
> the mediawiki/core team

There's no mediawiki core team… it's kind of anyone's responsibility to keep them in shape.